lua/tardis/screens/cl_screen_coordinates.lua

```diff
@@ -138,28 +138,43 @@ TARDIS:AddScreen("Destination", {id="coordinates", text="Coordinates", menu=fals
         local name = ""
         local pos = Vector(0,0,0)
         local ang = Angle(0,0,0)
-        Derma_StringRequest("New Location",
-        "Enter the name for the new location",
-        "New Location", 
-        function(text)
-            name = text
-            Derma_Query(
-                "Use current TARDIS position and rotation?\nSelecting 'No' will use the information on the inputs.",
-                "New Location",
-                "Yes",
-                function() 
-                        pos = ext:GetPos() 
-                        ang = ext:GetAngles() 
-                        if name == "" then name = "Unnamed" end
-                        TARDIS:AddLocation(pos,ang,name,map) 
-                        updatelist() 
-                end,
-                "No", 
-                function() 
+        local vortex = ext:GetData("vortex", false)
+
+        local request = "Enter the name for the new location"
+        if vortex then
+            request = request .. " (The info from the inputs would be used)"
+        end
+
+        Derma_StringRequest(
+            "New Location",
+            request,
+            "New Location",
+            function(text)
+                name = text
+                if vortex then
                     TARDIS:AddLocation(pos,ang,name,map)
-                    updatelist() 
-                end)
-        end)
+                    updatelist()
+                else
+                    Derma_Query(
+                        "Use current TARDIS position and rotation?\nSelecting 'No' will use the information on the inputs.",
+                        "New Location",
+                        "Yes",
+                        function()
+                            pos = ext:GetPos()
+                            ang = ext:GetAngles()
+                            if name == "" then name = "Unnamed" end
+                            TARDIS:AddLocation(pos,ang,name,map)
+                            updatelist()
+                        end,
+                        "No",
+                        function()
+                            TARDIS:AddLocation(pos,ang,name,map)
```

Comment:
This seems to always be 0,0 no matter what, what's the intended behaviour here?

Reply:
That's a mistake, but an old one.
It behaves absolutely the same way on "release" branch

I've opened an issue for this --> #624, we'll try to fix that soon
However, since this PR isn't the reason for this bug, I suggest to merge it now and make the fix separately

Reply:
That's a mistake, but an old one.
It behaves absolutely the same way on "release" branch

I've opened an issue for this --> #624, we'll try to fix that soon
However, since this PR isn't the reason for this bug, I suggest to merge it now and make the fix separately

@MattJeanes FYI

Reply:
Okay, sounds good I was thinking that would have also done this before too